databuilder/tables/beta/tpp.py

```diff
@@ -197,3 +198,69 @@ class ons_cis(EventFrame):
     visit_num = Series(int)
     is_opted_out_of_nhs_data_share = Series(bool)
     last_linkage_dt = Series(datetime.date)
+
+
+@table
+class isaric(EventFrame):
+    """
+    A subset of the ISARIC data.
+
+    These columns are deliberately all taken as strings while in a preliminary phase.
+    They will later change to more appropriate data types.
+
+    Descriptions taken from:
+    https://github.com/isaric4c/wiki/blob/d6b87d59a277cf2f6deedeb5e8c1a970dbb970a3/ISARIC/CCP_REDCap_ISARIC_data_dictionary_codebook.pdf
+    """
+
+    # Demographics
+    age = Series(str, description="Age")
+    age_factor = Series(str, description="TODO")
+    calc_age = Series(
+        str,
+        description="Calculated age (comparing date of birth with date of enrolment). May be inaccurate if a date of February 29 is used.",
+    )
+    sex = Series(str, description="Sex at birth.")
```

Comment:
Stylistic nit, but if you put a trailing comma on these arguments then Black would consistently format them on a separate line (as it does with the longer descriptions) and I think overall that might be more readable.

Reply:
That's a lot nicer. (Though this is the kind of thing I just expect Black should handle.)